src/main/java/owlmoney/logic/regex/RegexUtil.java

```diff
@@ -29,6 +29,18 @@ public static boolean regexCheckBankAmount(String input) {
         return input.matches(moneyRegex);
     }
 
+    /**
+     * Checks whether goal values input by user is within the 9 digit constraints with a max of 2 decimal places
+     * and value of at least 1.
+     *
+     * @param input The user input that is subject to Regex checking.
+     * @return the result of the check on whether it fulfills the criteria.
+     */
+    public static boolean regexCheckGoalsAmount(String input) {
```

Comment:
why is this regex the same as 

public static boolean regexCheckBankAmount(String input) {
        final String moneyRegex = "^\\s*(?=.*[0-9])\\d{1,9}(\\.\\d{1,2})?$";
        return input.matches(moneyRegex);
    }